global.json

```diff
@@ -1,6 +0,0 @@
-{
```

Comment:
Just a bump to discuss this change

Reply:
i read a bit into this and will add a global.json again. This should improve reproducibility at the cost of building inconvinience. I also upped the required version to the newest .NET Core SDK version.